src/controllers/v1/mentees.js

```diff
@@ -67,7 +73,11 @@ module.exports = class Mentees {
 
 	async reports(req) {
 		try {
-			const reports = await menteesService.reports(req.decodedToken.id, req.query.filterType)
+			const tenantCode = req.decodedToken.tenant_code
+			const organizationCode = req.decodedToken.organization_code
```

Comment:
@sumanvpacewisdom directly pass req.decodedToken.organization_code,req.decodedToken.tenant_code to method, do this change in all the places